### DISCUSSION TITLE
Feature Request(s): Inventory Grid Always-on/Count

### DISCUSSION THREAD

**Discussion by Jetbent** [runelite/runelite#15428] [Ideas]
Currently the Inventory-Grid plugin only displays when dragging an item in a player's inventory. 

However, without a grid available and many empty inventory spaces, it can be difficult to quickly determine how much open space a player has visually.

By offering a configuration option to always display the inventory grid, players can at a glance see what spaces they have available.

This can also make it helpful to know which locations in the player's inventory will be "dead" clicks or not. 

Similarly, offering an option to visually display a count of empty inventory spaces or not as a separate alternative configuration option would help for tasks that require a specific number of empty inventory slots in a more user friendly way.

### Current configuration options
![Config](https://user-images.githubusercontent.com/19807013/182209365-d2f0e421-7422-4093-aafb-f343be513e5c.png)

### Inventory grid only displays on drag
![Grid-on-drag](https://user-images.githubusercontent.com/19807013/182209372-7a4f0cb7-088b-498d-8011-4dd87f63a12c.png)

### Tough to gauge empty slots visually
![No-grid-default](https://user-images.githubusercontent.com/19807013/182209374-caa9e397-78cc-4fb6-aa27-65216b493ef3.png)

**Comment by mastersuleman** [comment]
I'd love this feature
